### PR TITLE
[v18.2.x] docs: add release notes for v18.2.0

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
## Issue being fixed or feature implemented

adds release notes

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
These changes should go into v18.2.x branch first and should be brought into develop via "merge master into develop" PR after v18.2 release.

File: doc/release-notes.md [hunk 1]
```diff
@@ -69,204 +69,47 @@ This release **does** include breaking changes. Please read below to see if they
...
-  stop-starting your node.
 
 Remote Procedure Call (RPC) Changes
 -----------------------------------
 
-Most changes here were introduced through Bitcoin backports mostly related to
-the deprecation of wallet accounts in DashCore v0.17 and introduction of PSBT
-format.
-
 ### The new RPCs are:
```
Comment (thephez) [inline doc/release-notes.md:76]: #4877 added the `getbalances` RPC if I'm not mistaken

File: doc/release-notes.md [hunk 2]
```diff
@@ -69,204 +69,47 @@ This release **does** include breaking changes. Please read below to see if they
...
+  provides information about what information is missing for that
+  input, including whether a UTXO needs to be provided, what pubkeys
+  still need to be provided, which scripts need to be provided, and
+  what signatures are still needed. Every input will also list which
+  role is needed to complete that input, and `analyzepsbt` will also
+  list the next role in general needed to complete the PSBT.
+  `analyzepsbt` will also provide the estimated fee rate and estimated
+  virtual size of the completed transaction if it has enough
+  information to do so.
+- `quorum listextended` is the cousin of `quorum list` with more enriched reply. By using `height` parameter RPC will list active quorums at specified height (or at tip if `height` is not specified.)
```
Comment (thephez) [inline doc/release-notes.md:90]:
```suggestion
- `quorum listextended` is the cousin of `quorum list` with a more enriched reply. By using the `height` parameter, the RPC will list active quorums at a specified height (or at the tip if `height` is not specified).
```

**Review by thephez — COMMENTED** [review]
Couple suggestions

**Review by thephez — DISMISSED** [review]
LGTM

**Review by UdjinM6 — DISMISSED** [review]
We used to do it this way https://github.com/dashpay/dash/pull/5129#pullrequestreview-1233306466. But utACK in case you want to do it like for #5128.

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**